code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -466,6 +467,7 @@
 	icon_state = "soda_dispenser"
 	base_icon_state = "soda_dispenser"
 	has_panel_overlay = FALSE
+	dispensed_temperature = 274 // cold enough that ice won't melt
```

Comment:
think the intention here is to be relative to 0C so

```suggestion
	dispensed_temperature = (T0C + 1) // cold enough that ice won't melt
```

Reply:
T0C + 1 would mean ice will melt

Reply:
ok

Reply:
this would mean it's at the exact temperature for ice to melt (reactions use <= or >= for their temperature checks depending on if it's a cold recipe or not)

Reply:
Why not just T0C?

Reply:
1. 273.15 (aka T0C) + 0.85 is 274

stop trolling me

Reply:
Ah, sorry! I'm not trolling!